<?xml version="1.0" encoding="UTF-8"?>
<testrun name="FunSetSuite" project="progfun-funsets" tests="10" started="10" failures="0" errors="1" ignored="0">
  <testsuite name="funsets.FunSetSuite" time="0.02">
    <testcase name="string take" classname="funsets.FunSetSuite" time="0.0"/>
    <testcase name="adding ints" classname="funsets.FunSetSuite" time="0.0"/>
    <testcase name="contains is implemented" classname="funsets.FunSetSuite" time="0.0"/>
    <testcase name="singletonSet" classname="1" time="0.001"/>
    <testcase name="union contains all elements" classname="funsets.FunSetSuite" time="0.001"/>
    <testcase name="intersect" classname="funsets.FunSetSuite" time="0.001"/>
    <testcase name="diff" classname="funsets.FunSetSuite" time="0.001"/>
    <testcase name="filter" classname="funsets.FunSetSuite" time="0.002"/>
    <testcase name="forall" classname="funsets.FunSetSuite" time="0.003"/>
    <testcase name="exists" classname="funsets.FunSetSuite" time="0.011">
      <error>org.scalatest.exceptions.TestFailedException
	at org.scalatest.Assertions$class.newAssertionFailedException(Assertions.scala:316)
	at funsets.FunSetSuite.newAssertionFailedException(FunSetSuite.scala:15)
	at org.scalatest.Assertions$class.assert(Assertions.scala:311)
	at funsets.FunSetSuite.assert(FunSetSuite.scala:15)
	at funsets.FunSetSuite$$anonfun$13$$anon$7.&lt;init&gt;(FunSetSuite.scala:151)
	at funsets.FunSetSuite$$anonfun$13.apply$mcV$sp(FunSetSuite.scala:148)
	at funsets.FunSetSuite$$anonfun$13.apply(FunSetSuite.scala:148)
	at funsets.FunSetSuite$$anonfun$13.apply(FunSetSuite.scala:148)
	at org.scalatest.FunSuite$$anon$1.apply(FunSuite.scala:1265)
	at org.scalatest.Suite$class.withFixture(Suite.scala:1974)
	at funsets.FunSetSuite.withFixture(FunSetSuite.scala:15)
	at org.scalatest.FunSuite$class.invokeWithFixture$1(FunSuite.scala:1262)
	at org.scalatest.FunSuite$$anonfun$runTest$1.apply(FunSuite.scala:1271)
	at org.scalatest.FunSuite$$anonfun$runTest$1.apply(FunSuite.scala:1271)
	at org.scalatest.SuperEngine.runTestImpl(Engine.scala:198)
	at org.scalatest.FunSuite$class.runTest(FunSuite.scala:1271)
	at funsets.FunSetSuite.runTest(FunSetSuite.scala:15)
	at org.scalatest.FunSuite$$anonfun$runTests$1.apply(FunSuite.scala:1304)
	at org.scalatest.FunSuite$$anonfun$runTests$1.apply(FunSuite.scala:1304)
	at org.scalatest.SuperEngine$$anonfun$org$scalatest$SuperEngine$$runTestsInBranch$1.apply(Engine.scala:260)
	at org.scalatest.SuperEngine$$anonfun$org$scalatest$SuperEngine$$runTestsInBranch$1.apply(Engine.scala:249)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:249)
	at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:326)
	at org.scalatest.FunSuite$class.runTests(FunSuite.scala:1304)
	at funsets.FunSetSuite.runTests(FunSetSuite.scala:15)
	at org.scalatest.Suite$class.run(Suite.scala:2303)
	at funsets.FunSetSuite.org$scalatest$FunSuite$$super$run(FunSetSuite.scala:15)
	at org.scalatest.FunSuite$$anonfun$run$1.apply(FunSuite.scala:1310)
	at org.scalatest.FunSuite$$anonfun$run$1.apply(FunSuite.scala:1310)
	at org.scalatest.SuperEngine.runImpl(Engine.scala:362)
	at org.scalatest.FunSuite$class.run(FunSuite.scala:1310)
	at funsets.FunSetSuite.run(FunSetSuite.scala:15)
	at org.scalatest.junit.JUnitRunner.run(JUnitRunner.scala:94)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)

</error>
    </testcase>
  </testsuite>
</testrun>
